addon/hifi-connections/base.js

```diff
@@ -11,16 +11,16 @@ import DebugLogging from '../mixins/debug-logging';
 
 
 /**
-* This is the base sound object from which other sound objects are derived. 
+* This is the base sound object from which other sound objects are derived.
 *
 * @class BaseSound
 * @constructor
 */
 
 let ClassMethods = Mixin.create({
   setup(config) {
-    this.config = config;
-    this.debugEnabled = config.debugEnabled;
+    this.config = config || {};
+    this.debugEnabled = this.config.debugEnabled || false;
   },
```

Comment:
i don't think our babel config actually supports this here so no change needed, but always looking for chances to evangelize the nullish coalescing operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
ooooooooo